control-plane-arm.json

```diff
@@ -100,22 +100,30 @@
                 "description": "Optional. A Slack channel by name with \"channel\": \"#other-channel\", or send a Slackbot message to a specific user with \"channel\": \"@username\". Fill this property to receive cluster related alerts through Slack. "
             }
         },
+        "oauthEnabled": {
+            "defaultValue": "false",
+            "type": "string",
+            "allowedValues": ["false", "true"],
+            "metadata": {
+                "description": "Enables OAuth based authentication and JWT based authorization for using Pipeline API. This field is not used if OAuth is enabled."
```

Comment:
I think the "this field is not used..." is not needed here

Reply:
good catch